extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -348,6 +356,27 @@ export class AmpStoryPage extends AMP.BaseElement {
     this.setPageDescription_();
   }
 
+  /** @private */
+  maybeSetPreviewDuration_() {
```

Comment:
Since this is about the advancement, should this live in the `page-advancement` file?
The page should not care about the preview mode, that's the advancement config's job.

https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/page-advancement.js#L204
The `forElement` method could return the expected configuration on the first call, removing the need to override it with a second one.

Reply:
Discussed offline, we thought it was better to keep it at the `amp-story-page` level, and fix the order in which `this.advancement_` is declared in there.